include/fluent-bit/flb_log.h

```diff
@@ -92,6 +92,12 @@ struct flb_log_cache {
     struct mk_list entries;         /* list for entries */
 };
 
+/* Simple structure to dispatch messages to the log collector */
```

Comment:
Revert this change so this structure is defined in `flb_log.c`

Reply:
I moved this and the prototype so I could write the benchmarking code and call `flb_log_construct` from there.
Thanks for the review, I'll do as requested.